Modules/Product/Http/Services/ProductService.php

```diff
@@ -220,12 +220,12 @@ public function arrangeProduct($request): array
             ->get();
 
         foreach ($products as $product) {
-            $productVariations = Product::find($product->id)->productVariations()->where('status', '1')->get();
+            $productVariations = Product::find($product->id)->with('productVariations')->where('status', '1')->get();
```

Comment:
You did it wrong. Get the productVariations when you get the products.
Like Product::with('productVariations')......
and then in foreach loop use it like
$productVariations = $product->productVariations()->where('status', '1')->get();